.github/dependabot.yaml

```diff
@@ -0,0 +1,43 @@
+version: 2
+updates:
+
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    labels:
+      - "area/dependencies"
+
+  # We maintain updates for most dependencies. This disables updates other than
+  # security ones.
+  - package-ecosystem: "cargo"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    labels:
+      - "area/dependencies"
+    open-pull-requests-limit: 0
+
+  - package-ecosystem: "gomod"
+    directory: "/tools"
+    schedule:
+      interval: "daily"
+    labels:
+      - "area/dependencies"
+    open-pull-requests-limit: 0
```

Comment:
We don't have any `Go` module dependencies in tools - why add these?

Reply:
I guess just as a safeguard in case we do add something, but it can be removed. I suppose chances are pretty small that we will add something that is not rust-based to our tooling code.